tests/test_rest.py

```diff
@@ -100,7 +100,7 @@ async def test_version_supported(
     responses: aioresponses,
     rest_client: Go2RtcRestClient,
     server_version: str,
-    expected_result: bool,
+    expected_result: bool,  # noqa: FBT001
```

Comment:
I'd disable this for all tests in pyproject.toml.

Example:
https://github.com/MartinHjelmare/aiomysensors/blob/2c0725cf0f8d51bcb7ea3a3ec9c0df348d6eb329/pyproject.toml#L74-L75

Reply:
Thanks Martin! Separate PR: https://github.com/home-assistant-libs/python-go2rtc-client/pull/9